.travis.yml

```diff
@@ -8,4 +8,4 @@ node_js:
   - '0.12'
   - '0.10'
 after_script:
-  - npm run coveralls
+  - npm test
```

Comment:
We already have `language: node_js`, it implies that `script` is `npm test`. This change causes `npm test` to be executed twice.
(Add it removes coverage results)